plugins/dev9ghzdrk/DEV9.cpp

```diff
@@ -135,6 +172,7 @@ s32 CALLBACK DEV9init()
 
 	FLASHinit();
 
+#ifdef _WIN32
```

Comment:
This mapping part is rather confusing because of the repeated ifdefs. I suggest having just one ifdef cover it.

Reply:
updated